frontend/src/lib/i18n/en.json

```diff
@@ -210,6 +210,13 @@
       "commentOnThisIssue": "Comment on this issue",
       "answerSaveError": "Saving answer failed",
       "answerDeleteError": "Deleting answer failed"
+    },
+    "allQuestions": {
+      "title": "Your opinions",
+      "warning": "You still have {nofUnasweredQuestions} unanswered questions. Your data won’t be shown in the Election Compass until you have answered all of them.",
+      "info": "You can see all of  your opinions below and edit them by tapping on them.",
+      "answerButton": "Answer this question.",
```

Comment:
I would avoid having a `.` inside the button's text, but also a matter of preference